Application/Frontend/pages/api/add_freind_request_to_user.ts

```diff
@@ -0,0 +1,42 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { MongoClient, ObjectId } from 'mongodb';
+import { getMongoDbUri } from '@/lib/dbConfig';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (req.method === 'POST') {
+    const { id, requestingUserId } = req.body; // Now also receiving the ID of the user sending the friend request
+
+    let client: MongoClient | null = null;
+
+    try {
+      client = new MongoClient(getMongoDbUri());
+      await client.connect();
+      const db = client.db('Accord');
+      const accountsCollection = db.collection("Test_friend_list_request_insertion");
+
+      // Assuming `id` is the recipient's user ID. Add `requestingUserId` to the recipient's pendingFriendRequest array.
+      const updateResult = await accountsCollection.updateOne(
+        { _id: new ObjectId(id) }, // Ensure the ID is correctly formatted for MongoDB
```

Comment:
You should not be using this default is from MongoDB but rather our one. However, if this collection does not have a unique ID field and does not need one, you could keep this as long as it works.